internal/data_test.go

```diff
@@ -131,27 +143,25 @@ func TestConvertData(t *testing.T) {
 			evals: []interface{}{int64(6)},
 		},
 	}
+	table := "testtable"
 	spSchema := ddl.CreateTable{
-		"testtable",
-		[]string{"a", "b", "c"},
-		map[string]ddl.ColumnDef{
+		Name: table,
+		Cols: []string{"a", "b", "c"},
+		Cds: map[string]ddl.ColumnDef{
 			"a": ddl.ColumnDef{Name: "a", T: ddl.Int64{}},
 			"b": ddl.ColumnDef{Name: "b", T: ddl.Float64{}},
 			"c": ddl.ColumnDef{Name: "c", T: ddl.Bool{}},
-		},
-		[]ddl.IndexKey{},
-		"",
-	}
-	srcSchema := pgTableDef{
-		map[string]pgColDef{
-			"a": pgColDef{id: "int8"},
-			"b": pgColDef{id: "float8"},
-			"c": pgColDef{id: "bool"},
+		}}
+	srcSchema := schema.Table{
+		ColDef: map[string]schema.Column{
+			"a": schema.Column{Type: schema.Type{Name: "int8"}},
+			"b": schema.Column{Type: schema.Type{Name: "float8"}},
+			"c": schema.Column{Type: schema.Type{Name: "bool"}},
 		}}
 	for _, tc := range multiColTests {
 		conv := buildConv(spSchema.Name, spSchema, srcSchema)
```

Comment:
Done.
In buildConv, we now get spanner table name from spTable and source DB table name from srcTable.

For the second question, I'm not sure I understand the question.
Codewise, the table and col name conversion happens on toddl i.e. conversion from source schema to spanner.
In principle it can happen anywhere because it is built up incrementally, but statement.go no longer calls into the tablemap functions.

Reply:
Thanks. You answered my question, "the table and col name conversion happens on toddl".